components/ord-service/src/main/java/com/sap/cloud/cmp/ord/service/filter/ResponseFormatFilter.java

```diff
@@ -25,6 +28,14 @@ public FormattedRequest(ServletRequest request) {
         @Override
         public String getQueryString() {
             String query = super.getQueryString();
+            String acceptHeader = super.getHeader(ACCEPT_HEADER);
+
+            boolean acceptsEverything = StringUtils.equals(acceptHeader, "*/*");
```

Comment:
Why using this package `org.apache.commons.codec.binary.StringUtils;` instead of `"*/*".equals(acceptHeader)`?

Reply:
obsolete